README.md

```diff
@@ -2,9 +2,19 @@
 
 Scaffolding for a [Cockpit](https://cockpit-project.org/) module.
 
+# Development dependencies
+
+On Debian/Ubuntu:
+
+    $ sudo apt install gettext nodejs npm make
+
+On Fedora:
+
+    $ sudo dnf install gettext nodejs npm make
```

Comment:
No `npm` please -- we have tools/node-modules which runs this in a container.

Reply:
Nonsense -- I was thinking `npm install`, but of course we also need it for `npm run`. Not strictly (we could call it directly as well), but our Makefile does `npm run watch`.